flink-queryable-state/flink-queryable-state-client-java/src/test/java/org/apache/flink/queryablestate/client/state/ImmutableAggregatingStateTest.java

```diff
@@ -53,20 +54,28 @@ public void setUp() throws Exception {
         aggrState = ImmutableAggregatingState.createState(aggrStateDesc, out.toByteArray());
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void testUpdate() throws Exception {
-        String value = aggrState.get();
-        assertEquals("42", value);
+    @Test
+    void testUpdate() {
+        assertThatThrownBy(
+                        () -> {
+                            String value = aggrState.get();
+                            assertEquals("42", value);
```

Comment:
All assertions should be migrated to AssertJ.

Reply:
Fixed, replaced `org.junit.Assert.assertEquals` with `org.junit.jupiter.api.Assertions.assertEquals`